.ci/install.sh

```diff
@@ -12,6 +12,33 @@ else
   exit 1
 fi
 
+INSTALL_DIR="$HOME/.fnm"
+
+# Parse Flags
+parse_args() {
+  POSITIONAL=()
```

Comment:
I can also guess we can drop the POSITIONAL arguments. No need for them here. Other than that, looks very very good.

I guess eventually you could create a binary that wraps `fnm` and does everything you script did:

```bash
w app # starts app, installs using fnm if needed
w npm # runs npm, installs using fnm if needed
w node # runs node, installs using fnm if needed
```

😄